src/Processor/ProductOptionValue/ReferenceEntityOptionValuesProcessor.php

```diff
@@ -0,0 +1,175 @@
+                'option' => $productOption,
+            ]);
+
+            if (!$productOptionValue instanceof ProductOptionValueInterface) {
+                /** @var ProductOptionValueInterface $productOptionValue */
+                $productOptionValue = $this->productOptionValueFactory->createNew();
+                $productOptionValue->setCode(ProductOptionManager::getOptionValueCodeFromProductOption($productOption, CreateUpdateDeleteTask::AKENEO_PREFIX . (string) $record['code']));
+                $productOptionValue->setOption($productOption);
+                $this->entityManager->persist($productOptionValue);
+            }
+
+            $this->updateProductOptionValueTranslations($productOptionValue, $attribute, $record);
+
+            $this->entityManager->flush();
+        }
+    }
+
+    private function updateProductOptionValueTranslations(
+        ProductOptionValueInterface $productOptionValue,
+        AttributeInterface $attribute,
+        array $record
+    ): void {
+        //Seems not to be customizable on Akeneo, but can be removed
+        $translations = $record['values']['label'] ?? [];
+
+        if (0 === \count($translations)) {
+            foreach ($this->getLocales() as $locale) {
+                $translations[] = [
+                    'locale' => $locale,
+                    'data' => \sprintf('[%s]', $record['code']),
+                ];
+            }
+        }
+
+        foreach ($translations as  $translation) {
+            $locale = $translation['locale'];
+
+            if (null === $translation['data']) {
+                $translation = \sprintf('[%s]', $record['code']);
+                $this->akeneoLogger->warning(\sprintf(
+                    'Missing translation on choice "%s" for option %s, defaulted to "%s"',
+                    $record['code'],
+                    $attribute->getCode(),
+                    $translation,
+                ));
+            }
+
+            $productOptionValueTranslation = $this->productOptionValueTranslationRepository->findOneBy([
+                'locale' => $locale,
+                'translatable' => $productOptionValue,
+            ]);
+
+            if (!$productOptionValueTranslation instanceof ProductOptionValueTranslationInterface) {
+                /** @var ProductOptionValueTranslationInterface $productOptionValueTranslation */
+                $productOptionValueTranslation = $this->productOptionValueTranslationFactory->createNew();
+                $productOptionValueTranslation->setTranslatable($productOptionValue);
+                $productOptionValueTranslation->setLocale($locale);
+
+                $this->entityManager->persist($productOptionValueTranslation);
+            }
+
+            $productOptionValueTranslation->setValue($translation['data']);
+        }
+    }
+
+    private function getLocales(): iterable
```

Comment:
IMO, this function shoundn't be here.

Reply:
fixed in commit bf87467577e17a7eb1ed6df554bb360c3514c478